worlds/sm64ex/docs/en_Super Mario 64 EX Advance Setup.md

```diff
@@ -0,0 +1,66 @@
+# Super Mario 64 EX Advanced Setup Guide
+
+## Required Software
+
+- Super Mario 64 US Rom (Japanese will work also. Europe and Shindou not supported)
+- Either of: 
+  - [sm64pclauncher](https://github.com/N00byKing/sm64pclauncher/releases) or
+  - Cloning and building [sm64ex](https://github.com/N00byKing/sm64ex) manually.
+
+
+NOTE: The above linked sm64pclauncher is a special version designed to work with the Archipelago build of sm64ex.
+You can use other sm64-port based builds with it, but you can't use a different launcher with the Archipelago build of sm64ex.
+
+## Building with Textures, Models and Patch Files
+
+# Installation via sm64pclauncher (For Windows)
+First, install [MSYS](https://www.msys2.org/) as described on the page.
+- DO NOT INSTALL INTO A FOLDER PATH WITH SPACES.
+- Do all steps, up to and including step 6.
+- Use the default install directory.
+Then follow the steps below.
+
+
+1. Go to the page linked for sm64pclauncher, and press on the topmost entry.
+3. Scroll down, and download the zip file.
+4. Unpack the zip file into an empty folder.
+5. Run the Launcher and press build.
+6. When prompted, set the location where you installed MSYS. Check the "Install Dependencies" Checkbox.
+7. Set the Repo link to `https://github.com/N00byKing/sm64ex` and the Branch to `archipelago` (Top two boxes). You can choose the folder (Second Box) at will, as long as it does not exist yet.
+8. Set the location for the Models Pack.
+9. Set the location for the Texture Pack.
+10. Point the Launcher to your Super Mario 64 US/JP Rom, and set the Region correspondingly.
+11. Copy the .patch file/s to the enhancements folder which is located in e.g. `C:/sm64pclauncher/apmario/enhancements`.
+  - NOTE: outside patches may break the game. Only the ones supplied with the repo are tested.
+12. Click on refresh patchlist. Click on the patches you want
+13. Set Build Options.
+  - Recommended: `-jn` where `n` is the Number of CPU Cores, to build faster.
+  - Recommended: Add EXTERNAL_DATA=1 to the build options.
+SM64EX will now be compiled. The Launcher will appear to have crashed, but this is not likely the case. Wait for it to finish.
+- There may be a problem if it takes longer than 10 minutes.
+
+
+After it's done, the Build list should have another entry titled with what you named the folder in step 7.
+
+NOTE: For some reason, starting the game for the first time always crashes the launcher. Just restart it.
+- If it still crashes, check if you typed the launch options correctly (described in "Joining a MultiWorld Game")

```

Comment:
"Joining a MultiWorld Game" doesn't exist on this page. Should this be a link to a different page?

Reply:
Yes this page is for recompiling the program is also a recommendation to use a fresh folder to prevent possible breaks to the original build that works

Reply:
I don't think you understood my point. What is "Joining a MultiWorld"? You can't reference a section that doesn't exist on this page and expect people to know where to go.

Reply:
oh thanks for catching that I will remove that part